### ISSUE TITLE
Can not reproduce the results.

### ISSUE THREAD

**Issue by DLwbm123** [pvjosue/LFMNet#2]
Hi! Thank you for releasing this dataset. I have tried to reproduce the results in your paper. However, the z-sum results of reconstructed confocal stacks are all black (zero). Could you please help me with this problem? Thanks!

**Comment by pvjosue** [comment]
Hi, 
It's hard to pinpoint the issue only with the information provided.
I would suggest to try again with the latest commits. In my setup it converges to a good solution with the current state of the repo.
If this doesn't work, please provide more info, about what you changed or what is your setup, so we can find the issue.
Cheers and thanks

**Comment by DLwbm123** [comment]
Hi! I have tried with the latest commits and it did not work. I have checked the tensorboard event files of both training and testing. The output images "z_proj_prediction" are all black(zero values).  The training loss and validation loss did not decrease during training. I have sent you the recorded tensorboard files by email. Please help me with this problem.  Thanks! The experimental settings in my experiments are as follow:
parser = argparse.ArgumentParser()
# Number of epochs
parser.add_argument('--epochs', type=int, default=200)
# Validate every n percentage of the data
parser.add_argument('--valEvery', type=float, default=0.4)
# Image indices to use for training and validation
parser.add_argument('--imagesToUse', nargs='+', type=int, default=list(range(0,30,1)))
# List of GPUs to use: 0 1 2 for example
parser.add_argument('--GPUs', nargs='+', type=int, default=None)
# Batch size
parser.add_argument('--batchSize', type=int, default=64)
# Perentage of the data to use for validation, from 0 to 1
parser.add_argument('--validationSplit', type=float, default=0.1)
# Bias initialization value
parser.add_argument('--biasVal', type=float, default=0.1)
# Learning rate
parser.add_argument('--learningRate', type=float, default=0.001)
# Use bias flag
parser.add_argument('--useBias', type=bool, default=True)
# Use skip connections flag
parser.add_argument('--useSkipCon', type=bool, default=False)
# User selected random seed
parser.add_argument('--randomSeed', type=int, default=None) 
# fov of input or neighboarhood around lenslet to reconstruct
parser.add_argument('--fovInput', type=int, default=9)
# nT number of lenslets to reconstruct simultaneously use at training time
parser.add_argument('--neighShape', type=int, default=3)
# Flag to use shallow or large U-net
parser.add_argument('--useShallowUnet', type=str2bool, default=True)
# Lower threshold of GT stacks, to get rid of autofluorescence
parser.add_argument('--ths', type=float, default=0.03)

**Comment by pvjosue** [comment]
I received your email, I'll have a look and get back to you.

**Comment by DLwbm123** [comment]
Thank you very much!

**Comment by pvjosue** [comment]
Good evening, 
I was able to reproduce the zero reconstructions that you where obtaining.
Seems that the problem was that I use batchNorm in the Unet, and bias is redundant as the batchNorm takes care of shifting the data distribution. The random initialization of the network's weights sometimes prevents the network from learning.

Would you mind trying with:
```python
# Use bias flag
parser.add_argument('--useBias', type=str2bool, default=False)
```

**Comment by DLwbm123** [comment]
Thanks! It works now.
